test/keeper/conditions/TransferNFTCondition.test.ts

```diff
@@ -128,11 +128,12 @@ describe('TransferNFTCondition', () => {
                 nftContractAddress: nftUpgradeable.address,
                 cap: BigNumber.from(100),
                 amount: nftAmount,
-                preMint: true
+                preMint: false
```

Comment:
don't we wanted to set `preMint` `true` by default?

Reply:
true or false is just a user option, here is false because we mint later

Reply:
I didn't see that it was just a test :)